app/models/repository_stock_value.rb

```diff
@@ -22,6 +22,44 @@ def low_stock?
     amount <= low_stock_threshold
   end
 
+  def self.add_filter_condition(repository_rows, join_alias, filter_element)
+    parameters = filter_element.parameters
+    if filter_element.operator == 'between'
+      return repository_rows if parameters['from'].blank? || parameters['to'].blank?
+    elsif parameters['value'].blank?
+      return repository_rows
+    end
```

Comment:
isn't this already handled on the fronted by the isBlank compted property?

Reply:
Well, we have the same number I think this is for some redundancy. Cause otherwise, the query will be not correct.

Reply:
alright